src/AppBundle/Entity/ConnectionRequest.php

```diff
@@ -72,6 +72,34 @@ class ConnectionRequest
      */
     protected $musicFriend = false;
 
+    /**
+     * @var boolean
+     *
+     * @ORM\Column(type="boolean")
+     */
+    protected $disqualified = false;
```

Comment:
As discussed earlier, write a migration for this. Just use `app/console doctrine:migrations:diff`.